controllers/generateConfigController.js

```diff
@@ -0,0 +1,190 @@
+const fs = require("fs");
+const { validEngines } = require("../utils/translationEnginesUtils");
+const { confirmUserAction, promptUserInput } = require("../utils/promptUtils");
+const { parsePath } = require("../utils/getConfigPath");
+const { listFilesOnDirectory } = require("../utils/listFiles");
+const { supportedLanguagesCodes } = require("../utils/supportedLanguagesUtils");
+
+const _promptTranslationEngines = () => {
+  const translationEngines = [...validEngines];
+  const translationEnginesToUse = [];
+
+  console.log(
+    "Will ask you for the translation engines you want to use. You will be able to change them later in the configuration file."
+  );
+
+  for (translationEngine of translationEngines) {
+    const shouldUseEngine = confirmUserAction(
+      `Do you want to use ${translationEngine} as translation engine? (y/n): `
+    );
+
+    if (shouldUseEngine) {
+      translationEnginesToUse.push(translationEngine);
+    }
+  }
+
+  return translationEnginesToUse;
+};
+
+const _promptBasePath = async () => {
+  let confirmedAction = false;
+  let pathFiles = [];
+  let basePath = "";
+
+  do {
+    let hasError = false;
+    basePath = promptUserInput(
+      'Base path for the translations files: b.e "src/localizations.": '
+    );
+
+    if (!basePath) {
+      console.log("The base path is required.\n");
+      continue;
+    }
+
+    const filesInPath = await listFilesOnDirectory(parsePath(basePath)).catch(
+      (err) => {
+        console.error(err.message);
+        console.log("\n-------------\n");
+        console.log(
+          "Please check that the path provided is correct and that you have the necessary permissions and try again.\n\n"
+        );
+        promptUserInput("Press enter to continue...\n\n");
+        console.clear();
+        hasError = true;
```

Comment:
Why not make the `continue` here?

Reply:
As it is on an arrow function if I make continue it will do it in the arrow function and not on the loop, this may cause an error since arrow function doesn't understand what is continue
![image](https://github.com/victor-heliomar/i18n-translation-generator/assets/66505715/fa1b1284-8ee5-46b4-a6dd-8aeaf1c42e09)

Reply:
ahh got it, thanks for the example.

So is it a matter of continuing to be on the same closure? Like the arrow function is inside another nested closure?

Reply:
sorry I didn't fully understand the question, can you explain me a little more please?